google-cloud-datastore/src/main/java/com/google/cloud/datastore/TransactionImpl.java

```diff
@@ -118,4 +127,9 @@ public void rollback() {
   public Datastore getDatastore() {
     return datastore;
   }
+
+  @Override
+  public ByteString getTransactionId() {
+    return transaction;
```

Comment:
The name was chosen before this code change, but please rename this to `transactionId`.

Reply:
Done.